### ISSUE TITLE
PhysX 5.0 - Available in 2022 ?

### ISSUE THREAD

**Issue by SparkyTD** [NVIDIAGameWorks/PhysX#517]
On the [PhysX website](https://developer.nvidia.com/physx-sdk) it is stated that the new SDK can be downloaded as part of an "Open Beta" in the NVIDIA Omniverse platform. I have downloaded the Omniverse app, but I haven't been able to find a download link to a C++ SDK. The way this is phrased on the website suggests that the SDK is downloadable in the open beta, which might be misleading. Downloading a pre-made sample app that *uses* PhysX 5 does not count as "downloading PhysX 5", and is completely irrelevant as far as developers are concerned.

Someone please tell me where I can sign up for this open beta, where I can download the C++ headers, static- and shared library files, or if that's not possible at this time, then please remove the misleading text from the website as soon as possible.

![image](https://user-images.githubusercontent.com/45818400/148850273-fb756629-81d0-4f30-9515-9523acae5878.png)

**Comment by RobertMtx** [comment]
I agree 100%. Very misleading. Apparently it is a gimmick to get more devs to installs Omniverse? I fell for it. Where is our PhysX 5 SDK promised at the NVidia website?

**Comment by monkeygame** [comment]
There will probably be no more SDKS released. Their goal is to roll out the NVIDIA Omniverse, which I've been experimenting with for a few days and which has a long, long way to go. My advice is: specialize, do what you're good at, and be great at it. SDK is the way to go.

**Comment by vinjn** [comment]
5.0 is here -> https://github.com/NVIDIA-Omniverse/PhysX
